### PR TITLE
Add Prettier

### PR DESCRIPTION
**Pull request by nathunsmitty**
## Types of changes

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [x] Refactor (change which changes the codebase without affecting its external behavior)
- [x] Non-breaking change (fix or feature that would causes existing functionality to work as expected)
- [ ] Breaking change (fix or feature that would cause existing functionality to __not__ work as expected)

## Purpose

There's this cool tool called [Prettier](https://github.com/prettier/prettier) that takes care of formatting for us. I thought we should use it.

## Approach

Basically just runs all our code through Prettier on a git pre-commit now.

## Learning

I followed the directions in their readme!

## Checklist

- [x] My branch follows the branch naming scheme of UCLA Radio, and can merge into `master` without error.
- [x] My code follows the code style of this project, and I have linted it to confirm this.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,14 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "dev": " concurrently --kill-others \"mongod\" \"grunt watch\" \"nodemon ./bin/www\" "
+    "dev": " concurrently --kill-others \"mongod\" \"grunt watch\" \"nodemon ./bin/www\" ",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
```
Comment (binerys) [inline package.json]: Change the "*.js" to also lint .jsx files.

Comment (nathunsmitty) [inline package.json]: 👍 see [295fff6](https://github.com/uclaradio/uclaradio/pull/129/commits/295fff694991a59879f0d572bc9d71a92d97c978)
